Projects/UOContent/Gumps/Go/GoGump.cs

```diff
@@ -7,12 +7,12 @@ namespace Server.Gumps
 {
     public class GoGump : Gump
     {
-        private static LocationTree Felucca;
-        private static LocationTree Trammel;
-        private static LocationTree Ilshenar;
-        private static LocationTree Malas;
-        private static LocationTree Tokuno;
-        private static LocationTree TerMur;
+        public static readonly LocationTree Felucca;
```

Comment:
I meant to do it here, but for all maps. Then we can delete the static ctor.
```suggestion
        Felucca = new LocationTree("felucca", Map.Felucca);
```

Reply:
Aaaah! Ok done.